src/main/pages/che-7/overview/proc_contributing-to-github-projects-in-hosted-che.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_contributing-to-github-projects-in-hosted-che.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="contributing-to-github-projects-in-hosted-che_{context}"]
+= Contributing to GitHub projects in Hosted Che
+
+This section describes how to contribute to GitHub projects in Hosted Che.
+
+[discrete]
+== Prerequisites
+
+* The link:https://github.com/eclipse/che-theia/tree/master/plugins/ssh-plugin[SSH Plug-in] is installed in the workspace.
+
+[discrete]
+== Procedure
+
+. Generate an SSH key pair with the link:https://github.com/eclipse/che-theia/tree/master/plugins/ssh-plugin[SSH Plug-in].
+
+. Upload the public key to the GitHub account following the link:https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account[Adding a new SSH key to your GitHub account] procedure.
+
```

Comment:
I'm missing a step that would show how to actually contribute. Maybe an example.

Reply:
That would be not Hosted Che specific, and we could add a link to the generic doc once it exists?

Reply:
I currently failed to install this plugin and test the procedure, maybe this should be done later, in another issue and PR?

Reply:
> That would be not Hosted Che specific, and we could add a link to the generic doc once it exists?

+1

> maybe this should be done later, in another issue and PR?

+1 File an issue, please.